### ISSUE TITLE
Export in file all records without pagination

### ISSUE THREAD

**Issue by muramidaza** [kartik-v/yii2-grid#994]
Is it possible to configure the component so that all records are output to a file when exported? For example, records are 2000. Records in the table are displayed with pagination, 500 records per page. But when exporting, all 2000 records should be included in the file.

**Comment by kartik-v** [comment]
Yes - but you need to use the kartik-v/yii2-export extension for that.
